File: Modul 5/Python/PRAK502-2310817220025-FirdaKhoirunisa.py
def hitung (nilai1, nilai2):
    hitungan = nilai1 - nilai2
    if hitungan < 0:
        return hitungan *-1
    else:
        return hitungan

def mutlak (angka):
    return abs(angka)

a, c, b, d = map(int, input().split())
Hasil = hitung(a, b) + hitung(c, d)
print(mutlak(Hasil))

File: Modul 5/C/PRAK502-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
    int hitungan = nilai1 - nilai2;
    if (hitungan < 0){
        return hitungan *= -1;
    }else{
        return hitungan;}
}
int mutlak(int angka){
    return abs(angka);
}

int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &c, &b, &d);

    int Hasil = hitung(a, b) + hitung(c, d);
    printf("%d", mutlak(Hasil));

    return 0;
}